lib/public/views/Logs/Create/index.js

```diff
@@ -25,6 +25,13 @@ const createScreen = (model) => {
     const errorMessages = model.logs.getErrorMessages()
         || [{ detail: 'There was a problem with posting the log, try again later' }];
 
+    const preview = h('', [
```

Comment:
Shouldn't this be an arrow function instead of a variable?

Reply:
Done